cli/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+
+    This program is licensed to you under the Apache License Version 2.0,
+    and you may not use this file except in compliance with the Apache License Version 2.0.
+    You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the Apache License Version 2.0 is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.sonatype.nexus</groupId>
+  <artifactId>nexus-platform-cli</artifactId>
```

Comment:
What is 'cli' supposed to stand for?  The only thing that comes to mind for me is command line interface.

Reply:
Should the artifact name be more closely aligned with the repo name circleci-nexus-orb?

Reply:
Since this is a cli specific to CircleCI, I wouldn't use org.sonatype.nexus:nexus-platform-cli (even if we don't plan on pushing the RSO). Instead it should indicate it's purpose. I've been leaning towards describing this in the groupId, i.e. `org.sonatype.nexus.ci.jenkins` - https://github.com/sonatype-nexus-community/jenkins-nexus-notifier-plugin/blob/master/pom.xml#L26

Reply:
Since this is a cli specific to CircleCI, I wouldn't use org.sonatype.nexus:nexus-platform-cli (even if we don't plan on pushing the RSO). Instead it should indicate it's purpose. I've been leaning towards describing this in the groupId, i.e. org.sonatype.nexus.ci.jenkins - https://github.com/sonatype-nexus-community/jenkins-nexus-notifier-plugin/blob/master/pom.xml#L26

Reply:
groupId -> org.sonatype.nexus.ci.circleci